recipe/patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch

```diff
@@ -54,7 +54,7 @@ index 97973bc..1340ad7 100644
          def find_library(name):
              # See issue #9998
 -            return _findSoname_ldconfig(name) or \
--                   _get_soname(_findLib_gcc(name) or _findLib_ld(name))
+-                   _get_soname(_findLib_gcc(name)) or _get_soname(_findLib_ld(name))
```

Comment:
this may have some change, I'll figure this out.

Reply:
Have a look at 3.8 branch's patch. That might help. https://github.com/conda-forge/python-feedstock/blob/3.8/recipe/patches/0013-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch

Reply:
Yeah, that confirmed it. I think this is fine.